apps/meteor/tests/e2e/omnichannel/omnichannel-contact-center.spec.ts

```diff
@@ -81,10 +81,6 @@ test.describe('Omnichannel Contact Center', () => {
 	test.afterAll(async ({ api }) => {
 		// Remove added contacts
 		await Promise.all([api.delete(`/livechat/visitor/${NEW_CONTACT.token}`), api.delete(`/livechat/visitor/${EXISTING_CONTACT.token}`)]);
-
-		if (IS_EE) {
-			await api.post('method.call/livechat:removeCustomField', { message: NEW_CUSTOM_FIELD.field });
-		}
```

Comment:
No tear down for the custom field here?

Reply:
I removed the 'teardown' because the test is not creating this custom field in the `before` hook. So there's nothing to delete.

That said, it might be useful removing the call to `livechat/custom.field` from the `before` hook. Because it's not being used in the tests from this file. Another detail is that the endpoint being called on `before` does not create a custom field, it adds the custom field to a visitor/contact. As it is currently implemented, it's throwing an 400 bad request because the endpoint is not properly used. But since it's not using the response for anything, it doesn't show in the tests.